src/ansys/mechanical/env/mechanical-env

```diff
@@ -122,8 +122,15 @@ export MWUSER_DIRECTORY
 MWDONT_XCLOSEDISPLAY=1
 export MWDONT_XCLOSEDISPLAY
 
-# Set dynamic library preload
-LD_PRELOAD=libstdc++.so.6.0.28
+# Set dynamic library preload from Ansys unified install
+AnsLibStdCpp="$(find ${!awp_root}/tp/ -maxdepth 2 -type f -name libstdc++.so.6.\* 2>/dev/null | head -1)"
```

Comment:
mechanical-env knows the product version number already, so we should be able to pick the correct libstdc++ based on that. I don't see a reason to do any bash scripting here.  One of my goals for mechanical-env is to be very readable so that users who want to use mechanical in another context (other than pymechanical) could read this script and know what variables they need to set.